src/sql/helpers.js

```diff
@@ -64,26 +64,44 @@ const jsonbObjectAgg = (aggregationColumnKey, nestedFields, aggregationJsonKey,
  * For this example:
  * - rootFields: ['id', 'name'],
  * - nestedFields: ['pipeline_arn', 'hash]
- * - aggregationColumnKey: 'pipeline_type'
- * - aggregationJsonKey: 'pipelines'
+ * - collapseKey: 'pipeline_type'
+ * - collapseKeyNewName: 'pipelines'
  *
  * Note it also camelcases all the keys returned
  */
-const aggregateIntoJson = (
+const collapseKeysIntoObject = (
   originalQuery,
   rootFields,
-  nestedFields,
-  aggregationColumnKey,
-  aggregationJsonKey,
+  fieldsToCollapse,
+  collapseKey,
+  collapseKeyNewName,
```

Comment:
rename `collapseKeyNewName` to `collapsedKeyNewName` to match the name in `collapseKeyIntoArray` function